libraries/apollo-api/src/commonMain/kotlin/com/apollographql/apollo3/api/http/DefaultHttpRequestComposer.kt

```diff
@@ -199,7 +199,7 @@ class DefaultHttpRequestComposer(
         }
         append(it.key.urlEncode())
```

Comment:
Should we do it here too?

Reply:
👍 there's no reason not to!

Reply:
Well, since this is common code, we're hoping that Android/JS can deal with the '+' sign like iOS 🤞 . If not, we'll have to expect/actual this. I'm not sure what the internet is saying about that. Is it generally accepted?

Reply:
I've always assumed `+` was how to treat spaces in URLs but apparently [it's complicated](https://stackoverflow.com/a/47188851/15695) 😅.

Maybe we should use `%20` instead to stay on the safe side? Also I'll move the test to `commonMain` so at least we know that it's not rejected by JVM and JS.